.github/workflows/close-stale.yml

```diff
@@ -0,0 +1,20 @@
+name: 'Close stale issues'
```

Comment:
See https://github.com/vercel/pkg/issues/1065#issuecomment-804915421 . I personally don't like automatic closing/locking of PRs/issues.

Reply:
Answered